api_blogs/lib/api_blogs/blog.ex

```diff
@@ -103,6 +120,11 @@ defmodule ApiBlogs.Blog do
     User.changeset(user, attrs)
   end
 
+  def extract_id(conn) do
```

Comment:
Se essa função for apenas utilizada nesse módulo é legal coloca-la como privada. Agora, caso você a utilize em outors lugares, cria um módulo chamado Helpers ou Utils e a deixa como pública. Assim você conseguirá utiliza-la em vários lugares sem precisar definir em todo os módulos.

Reply:
O ideal é que ela seja usada só nesse módulo mesmo, mas para isso preciso refatorar o `user_controller` para passar a lógica para o contexto, pois a função está sendo chamada lá. Vou fazer isso no próximo commit e tornar a função privada.